src/serialize.h

```diff
@@ -901,10 +901,9 @@ class CSizeComputer
 protected:
     size_t nSize;
 
-    const int nType;
     const int nVersion;
 public:
-    CSizeComputer(int nTypeIn, int nVersionIn) : nSize(0), nType(nTypeIn), nVersion(nVersionIn) {}
+    CSizeComputer(int nVersionIn) : nSize(0), nVersion(nVersionIn) {}
```

Comment:
Should be `explicit`? :-)

Reply:
yes, that's good practice for one-argument constructors